string name
float64 r_effort
float64 l_effort

